clients/githubrepo/graphql.go

```diff
@@ -152,6 +173,24 @@ func pullRequestFrom(data *graphqlData) []clients.PullRequest {
 	return ret
 }
 
+func releasesFrom(data *graphqlData) []clients.Release {
+	releases := make([]clients.Release, 0)
+	for _, r := range data.Repository.Releases.Nodes {
+		release := clients.Release{
+			TagName: string(r.TagName),
+			Assets:  make([]clients.ReleaseAsset, 0),
```

Comment:
nit: no need for this. you can append to nil arrays (unless you want to preallocate the capacity here with `len(r.ReleaseAssets.Nodes)`

Reply:
Done.